internal/region/region.go

```diff
@@ -134,3 +177,13 @@ func (r Region) WebDriverBaseURL() string {
 	meta := lookupMeta(r)
 	return meta.WebdriverBaseURL
 }
+
+func (r Region) Credentials() iam.Credentials {
+	meta := lookupMeta(r)
+
+	if meta.Credentials.IsSet() {
```

Comment:
Just nice to have this info, since the function `lookupMeta()` is very generic and the intent is not immediately clear (outside of this PR):
```suggestion
        // check if there are any region specific credentials first
	if meta.Credentials.IsSet() {
```